ci-operator/step-registry/ibmcloud/deprovision/resourcegroup/ibmcloud-deprovision-resourcegroup-commands.sh

```diff
@@ -48,6 +48,18 @@ function run_command_with_retries() {
 }
 
 ibmcloud_login
+hasError=0
+#OCP-60816 - [IPI-on-IBMCloud] install cluster under BYON with a different resource group
+#after destroyed the cluster, check the pre resource group is existed.
+if [ -s "${SHARED_DIR}/ibmcloud_cluster_resource_group" ]; then
+    resource_group=$(cat "${SHARED_DIR}/ibmcloud_cluster_resource_group")
+    ${IBMCLOUD_CLI} resource group ${resource_group}
+    if [ $? != 0 ]; then
```

Comment:
I do not think need this if condition, if the above command return non-zero, the whole step would exit as failure directly, would never come to this if condition.